src/app/shell/personal-cabinet/parent/create-application/create-application.component.html

```diff
@@ -10,7 +10,7 @@ <h3 class="wrapper-title">ЗАЯВА НА ГУРТОК</h3>
 
         <label class="step-label">Виберіть дитину<span class="step-required">*</span></label>
         <mat-form-field appearance="none">
-          <mat-select disableOptionCentering panelClass="dropdown-panel" class="step-input"
+          <mat-select (selectionChange)="checkAbilityToChildApply()" disableOptionCentering panelClass="dropdown-panel" class="step-input"
```

Comment:
if the user cannot apply new application because there is an active one then show please a warning message 
I think it could be displayed as a validation (but not be a real validation)
Just add a red text-hint below and add a red border to the dropdown